.github/workflows/tests-integration.yml

```diff
@@ -0,0 +1,17 @@
+name: Build and Test Packages
+on: [push]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Foundry
+        uses: onbjerg/foundry-toolchain@v1
```

Comment:
Could you change it from onbjerg/foundry-toolchain@v1 to `foundry-rs/foundry-toolchain@v1` - I know onbjerg created foundry so it mostly fine but safer to use the official version

Reply:
On it